vibin/device_resolution.py

```diff
@@ -311,7 +311,6 @@ def _determine_amplifier_device(
                 device
                 for device in devices
                 if "MediaRenderer" in device.device_type
-                and device != streamer_device
```

Comment:
This condition (`device != streamer_device`) is duplicated in the block below and actually makes that block redundant. I assume it's safe to remove.

Reply:
Yep agreed, this should be safe.

I currently can't validate it with my StreamMagic + Hegel setup because UPnP discovery is currently broken (for reasons); but it looks fine, and everything still works as expected with `--amplifier`.